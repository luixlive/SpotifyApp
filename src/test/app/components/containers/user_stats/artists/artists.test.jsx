import _ from 'lodash';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import React from 'react';
import renderer from 'react-test-renderer';

import {
  artists as componentProps,
} from './../../../../../test_utils/components_props';
import initialState from './../../../../../test_utils/initial_state';
import Artists
/* eslint-disable-next-line max-len */
  from './../../../../../../app/components/containers/user_stats/artists/artists';

describe('App Components - Artists', () => {
  describe('Snapshots', () => {
    it('renders', () => {
      const mockStore = configureStore();
      const props = _.cloneDeep(componentProps);
      const store = mockStore(initialState);
      const rendered = renderer.create((
        <Provider store={store}>
          <Artists {...props} />
        </Provider>
      )).toJSON();
      expect(rendered).toMatchSnapshot();
    });
  });
});
