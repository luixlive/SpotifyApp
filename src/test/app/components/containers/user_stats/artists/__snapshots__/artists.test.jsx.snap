// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Components - Artists Snapshots renders 1`] = `
<div
  className="ui basic segment"
>
  <div
    className="ui grid"
  />
</div>
`;
